core/config/config.go

```diff
@@ -96,6 +96,15 @@ func GetEnvironment() string {
 	return env
 }
 
+// Path returns path to the yaml configurations
```

Comment:
Its not, but we only deal with yaml configurations as of now. Updating the comment with just 'configurations'